packages/cli/src/external-hooks.ts

```diff
@@ -35,24 +109,15 @@ export class ExternalHooks {
 		};
 	}
 
-	async init(): Promise<void> {
-		if (this.initDidRun) {
-			return;
-		}
-
-		await this.loadHooksFiles();
-
-		this.initDidRun = true;
-	}
-
-	private async loadHooksFiles() {
+	async init() {
```

Comment:
How do we guarantee that by the time we inject `ExternalHooks` anywhere it's already been initialized? If we ever fail to initialize it (e.g. remove the call by accident), then external hooks have zero registrations so they won't call anything but we won't be aware of this.

Reply:
We don't. This service is initialized in `init` of various commands, just like `init` for `BinaryDataService`, or `DataDeduplicationService`, etc.
I agree that we need a test to ensure that all relevant commands call these inits at the very start. but, I don't think we should keep these `isInitialized` flag in so many of these services.